Description: Ignore Debian-specific packaging dirs debian/ and .pc/ in bundlebuilder.py
Author: Jonas Smedegaard <dr@jones.dk>
Last-Update: 2010-02-08
--- a/src/sugar/activity/bundlebuilder.py
+++ b/src/sugar/activity/bundlebuilder.py
@@ -36,7 +36,7 @@
 from sugar.bundle.activitybundle import ActivityBundle
 
 
-IGNORE_DIRS = ['dist', '.git']
+IGNORE_DIRS = ['dist', '.git', 'debian', '.pc']
 IGNORE_FILES = ['.gitignore', 'MANIFEST', '*.pyc', '*~', '*.bak', 'pseudo.po']
 
 
